common/src/main/java/com/wynntils/features/user/CustomNametagRendererFeature.java

```diff
@@ -104,13 +100,12 @@ private static void getItemComponent(NametagRenderEvent event, ItemStack itemSta
     }
 
     private static void addAccountTypeNametag(NametagRenderEvent event) {
-        for (AccountType type : AccountType.values()) {
-            if (type.getComponent() == null) continue;
+        WynntilsUser user = UserInfoModel.getUser(event.getEntity().getUUID());
+        if (user == null) return;
+        AccountType accountType = user.accountType();
+        if (accountType.getComponent() == null) return;
 
-            if (UserInfoModel.isAccountType(event.getEntity().getUUID(), type)) {
-                event.addInjectedLine(type.getComponent());
-            }
-        }
+        event.addInjectedLine(accountType.getComponent());
     }
```

Comment:
This was refactored due to only a single account type per account, so we don't need to loop through all of them.